app/src/main/java/com/android/sample/ui/components/SearchBar.kt

```diff
@@ -0,0 +1,32 @@
+package com.android.sample.ui.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Search
```

Comment:
Can you please use materials3 instead

Reply:
The icons are part of the material package there is no equivalent but we use Icon() from material3 which keeps it coherent